packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2153,6 +2154,9 @@ class RenderCustomPaint extends RenderProxyBox {
     markNeedsLayout();
   }
 
+  /// Whether to require that the painting output be held in the raster cache.
+  bool forceRasterCache;
```

Comment:
I would name this `isComplex`. The documentation is good though. Maybe add some more color describing what the raster cache is and what the implications are of being held there?